app/react/Library/components/HiddenColumnsDropdown.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { DropdownList } from 'app/Forms';
```

Comment:
To avoid a bubble up of not being able to use TS because of this component, please, just migrate this file to TS and ts_ignore (or however it's used) this subcomponent.